docs/rules/no-ternary.md

```diff
@@ -8,9 +8,9 @@ var foo = isBar ? baz : qux;
 
 ## Rule Details
 
-The `no-ternary` rule aims to disallow the use of ternary operators.
+This rule disallows ternary operators.
 
-The following patterns are considered problems:
+Examples of **incorrect** code for this rule:
```

Comment:
Ouch, forgive me for asking, but could you rewrite the examples to avoid the naming conflicts like `baz` and `baz()`

To reduce from 3 to 2 could drop the middle example but transplant those function calls into the function example.